_posts/2014-07-25-order.md

```diff
@@ -190,12 +190,12 @@ requested_ship_date
   Whether or not this date becomes the Target Ship Date depends on the merchant's target ship date cutoff time and the time
   the order is placed.
   
-  Format: `YYYY-MM-DD`
+  Format: `YYYY-MM-DD` or an ISO8601 timestamp.
```

Comment:
The final version in master was changed to not have any effect on requested_ship_date, only delayed_ship_date.
```suggestion
  Format: `YYYY-MM-DD`
```